proto/livekit_analytics.pb.go

```diff
@@ -2,9 +2,9 @@
 // versions:
 // 	protoc-gen-go v1.27.1
 // 	protoc        v3.17.3
-// source: livekit_analytics.proto
+// source: proto/livekit_analytics.proto
 
-package livekit
+package proto
```

Comment:
We have a gotcha: an unfortunate trade-off.

With `--go_out . --go_opt=paths=source_relative`, which is the only pattern that puts `livekit_*.pb.go` in this location, the package name must match the directory name.

That is, if we wish to have Go files at `proto/livekit_*.pb.go`, they must have `package proto`.

If we wish them to have `package livekit`, we must indicate `go_package = something/livekit` in the `*.proto` files and then we will end up with `livekit/livekit_*.pb.go`.

Either way we will break some kind of backward compatibility.  Either the `livekit_*.pb.go` package names must change, or their paths must change.

Reply:
IMO, of these alternatives the prettier one is to end up with `package livekit` and to put all the files in `livekit/livekit_*.pb.go`.

However, before changing it again I will wait for a discussion and the conclusion of my test in another project, of importing these types.